fastparse/shared/src/main/scala/fastparse/sequence/VectorReprOps.scala

```diff
@@ -0,0 +1,34 @@
+package fastparse.sequence
+import fastparse.utils._
+import scala.reflect.ClassTag
+
+case class VectorReprOps[T]()(implicit evidence: ClassTag[T]) extends fastparse.utils.ReprOps[T, Vector[T]] {
+  type U = Vector[T]
+  val U = Vector
+  override def apply(input: U, i: Int) = input(i)
+  override def slice(input: U, start: Int, end: Int) = input.slice(start, end)
+  override def length(input: U) = input.length
+
+  override def fromArray(input: Array[T]) = input.toVector
+  override def fromSeq(input: Seq[T]) = input.toVector
+  override def fromSingle(input: T) = U(input)
+  override def toArray(input: U) = input.toArray
+  override def flatten(input: Seq[U]) = input.flatten.toVector
+  override def prettyPrint(input: U) = input.mkString("[", ", ", "]")
+  override def literalize(input: U): String = prettyPrint(input)
+  override def errorMessage(input: fastparse.utils.ParserInput[T, U], expected: String, idx: Int): String = {
```

Comment:
It's not clear to me what this `errorMessage` implementation is doing; let's have one unit test that checks the error message to make sure it's what you expect, and that readers of the code could use to see what the final output of this code is meant to be